### PR TITLE
Migrate investment features to cypress

### PR DESCRIPTION
**Pull request by PippoRaimondi**
## Description of change

This PR will be responsible to migrate all investment project features to cypress

## Test instructions

yarn test:e2e:dit
 
- [ ] Has the branch been rebased to develop?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/lib/urls.js [hunk 1]
```diff
@@ -101,6 +101,7 @@ module.exports = {
     index: url('/companies'),
     interactions: createInteractionsSubApp('/companies', '/:companyId'),
     orders: url('/companies', '/:companyId/orders'),
+    investment: url('/companies', '/:companyId/investments'),
```
Comment (mcbhenwood) [inline src/lib/urls.js]: Fiddly though it is, I do think that this property should have a plural name to match the names ("interactions", "orders", etc) of other properties.

That may mean that you call it `company_investments` or (preferably) that you assign a new name to the existing `investments` property which follows.

File: test/end-to-end/cypress/specs/DIT/collection-spec.js [hunk 2]
```diff
@@ -25,4 +27,24 @@ describe('Collection', () => {
       checkCollection()
     })
   })
+
+  describe('investment interaction', () => {
+    before(() => {
+      cy.visit(investments.projects.interactionCollection('721e2a04-21c3-4172-a321-4368463a4b2d'))
```
Comment (mcbhenwood) [inline test/end-to-end/cypress/specs/DIT/collection-spec.js]: Is it worth us collecting these fixture UUIDs in a central module somewhere then referring to them like:
```
const { interactionCollectionUUID } = require ('superTestFixtureUUIDs')
...
  cy.visit(investments.projects.interactionCollection(interactionCollectuionUUID))
```

(optional)

Comment (PippoRaimondi) [reply test/end-to-end/cypress/specs/DIT/collection-spec.js]: yes, this is coming as part of the refactor I want to do... I will move the fixtures folder up a level and update functionals and e2e to use these same fixtures (want to do this in a single PR)...

File: src/lib/urls.js (hunk 1 above)
Comment (PippoRaimondi) [reply src/lib/urls.js]: agreed, will sort this out!

**Pull request merged**